tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1381,12 +1377,6 @@ def state_size(self):
   def output_size(self):
     return self._num_units
 
-  def _linear(self, args, kernel, bias):
-    out = math_ops.matmul(args, kernel)
-    if not self._layer_norm:
-      out = nn_ops.bias_add(out, bias)
-    return out
-
   def _norm(self, inp, scope):
```

Comment:
can you avoid using scope= anywhere?  in TF 2.0, scopes are going away -- as is tf.get_variable.  you may need to copy some of the impl in layer_norm into this file.  see also pr #15861.